src/main/java/com/microsoft/aad/msal4j/TenantProfile.java

```diff
@@ -0,0 +1,20 @@
+package com.microsoft.aad.msal4j;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+
+import java.util.Map;
+
+@Accessors(fluent = true)
+@Getter
+@Setter
+public class TenantProfile extends Account implements ITenantProfile{
```

Comment:
Why tenant Profile extends Account ?

Reply:
A tenant profile is pretty much the same as an account, except that we need to know the local UID in addition to the home UID, and I based the extend/implement structure of the new classes off of the  implementation in Android.